spring-boot-webflux/src/main/java/com/cogrammer/service/ExampleService.java

```diff
@@ -1,34 +1,31 @@
 package com.cogrammer.service;
 
+import java.time.Duration;
+
 import com.cogrammer.dto.Cities;
 import com.cogrammer.dto.City;
 import com.cogrammer.dto.Message;
 import com.cogrammer.repository.CityRepository;
 import lombok.RequiredArgsConstructor;
-import org.springframework.stereotype.Service;
 import reactor.core.publisher.Mono;
 
+import org.springframework.stereotype.Service;
+
 @Service
 @RequiredArgsConstructor
 public class ExampleService {
 
-    private final CityRepository repository;
+	private final CityRepository repository;
 
-    public Mono<Message> blockingHello() {
-        return Mono.fromSupplier(() -> {
-            try {
-                Thread.sleep(100);
-                return new Message("Hello!");
-            } catch (InterruptedException e) {
-                throw new RuntimeException(e);
-            }
-        });
-    }
+	public Mono<Message> delayedHello() {
+		return Mono.just(new Message("Hello!"))
```

Comment:
The goal of the `blockingHello` method was to intentionally simulate the long-running method call (e.g blocking I/O [this might be done on the other hand using a worker threads pool although]) in a reactive environment. I don't think you're simulating the same behavior with `delayElement` call (since the method call itself is not blocking but you're adding an artificial delay)

Reply:
Fair enough. In this case you probably should schedule this work on a `boundedElastic` Scheduler, which is after the all the equivalent of what's done already in the other sample: https://github.com/tomekzar/quarkus-vs-spring-boot-performance/blob/b989f9d229b8c3a4c44d159e229eba4d64fa9c42/quarkus-jackson-mutiny/src/main/java/com/cogrammer/resource/ExampleResource.java#L31

Reply:
The rule is to *never* block the schedulers that are doing I/O. If you need to perform a blocking call, you can offload it to a blocking friendly scheduler (e.g. `Schedulers.boundedElastic()` or a custom one).

See https://youtu.be/0rnMIueRKNU?t=1638

Reply:
The fix is now introduced on master branch, tests will be executed once more and the results will be updated. Thanks for finding this issue.

```
@GetMapping(path = "/hello")
    public Mono<Message> hello() {
        return service.blockingHello()
                .subscribeOn(Schedulers.boundedElastic());
    }
```

Reply:
Thanks!